core/lib/scmi-server/include/fwk_assert.h

```diff
@@ -0,0 +1,39 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, Linaro Limited
+ *
+ */
+
+#ifndef SCMI_SERVER_FWK_ASSERT_H
+#define SCMI_SERVER_FWK_ASSERT_H
+
+#include <assert.h>
+#include_next <fwk_assert.h>
```

Comment:
Is this a temporary workaround?

Reply:
I don't think so. I think it will stay.
I can try to move this code in SCP-firmware arch/none/opee/include to get it maintained (and possibly removed one day) from there. Makes sense.

Reply:
OK, thanks. The optee part of the patch seems to suggest that it has something to do with OP-TEE adaptations.

Reply:
I've not removed this header file yet. It's planned but I prefer to be sure SCP-fmw is OK to host that wrapper header before doing so.

Reply:
OK